packages/multichain/src/scope/assert.test.ts

```diff
@@ -0,0 +1,208 @@
+import { JsonRpcError } from '@metamask/rpc-errors';
+
+import { assertScopeSupported, assertScopesSupported } from './assert';
+import * as Supported from './supported';
+import type { ScopeObject } from './types';
+
+jest.mock('./supported', () => ({
+  isSupportedScopeString: jest.fn(),
+  isSupportedNotification: jest.fn(),
+  isSupportedMethod: jest.fn(),
+}));
+const MockSupported = jest.mocked(Supported);
+
+const validScopeObject: ScopeObject = {
+  methods: [],
+  notifications: [],
+  accounts: [],
+};
+
+describe('Scope Assert', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
```

Comment:
This has been happening automatically since https://github.com/MetaMask/core/pull/4417, so it is not needed:

``` suggestion
```

Reply:
https://github.com/MetaMask/core/pull/4784/commits/18d1112b95e756334fbf06d7dd5b9ec24a8783f7